src/plugins/analysis/linter/test/test_source_code_analysis.py

```diff
@@ -32,45 +32,22 @@ def stub_plugin(test_config, monkeypatch):
     return AnalysisPlugin(MockAdmin(), test_config, offline_testing=True)
 
 
-@pytest.mark.parametrize('mime_and_type', [
-    ('Python script, ASCII text executable', 'python'),
-    ('python3 script, ASCII text executable', 'python'),
-    ('Bourne-Again shell script, UTF-8 Unicode text executable', 'shell')
-])
-def test_determine_script_type_file_type(mime_and_type, stub_plugin, test_object):
-    mime, script_type = mime_and_type
-    test_object.processed_analysis['file_type'] = {'full': mime}
-
-    assert stub_plugin._determine_script_type(test_object) == script_type
-
-
 @pytest.mark.parametrize('shebang_and_type', [
-    (b'#!/usr/bin/env python3', 'python'),
-    (b'#!/usr/bin/python', 'python'),
-    (b'#!/bin/bash', 'shell'),
-    (b'#!/bin/sh', 'shell')
+    (b'#!/usr/bin/env python3', b'testfile1:4 lines(4 sloc)\r\n  type:Text\r\n  mime type:text/plain\r\n  language:Python\r\n', 'python'),
```

Comment:
where did all those carriage returns ("\r") come from? I couldn't find any when I tested it :thinking: